pillar/heroku/discussions.sls

```diff
@@ -180,7 +181,11 @@ heroku:
     OPEN_DISCUSSIONS_BASE_URL: {{ env_data.OPEN_DISCUSSIONS_BASE_URL }}
     OPEN_DISCUSSIONS_COOKIE_DOMAIN: {{ env_data.OPEN_DISCUSSIONS_COOKIE_DOMAIN }}
     OPEN_DISCUSSIONS_COOKIE_NAME: {{ env_data.OPEN_DISCUSSIONS_COOKIE_NAME}}
+    {% if env_data.OCW_NEXT_URL is defined %}
```

Comment:
I think this would need to be `if env_data.get('OCW_NEXT_URL')` as the `defined` test is for a given variable, whereas this is using attribute lookup.

Reply:
cool, will change that